Robot_Adapter/Delete/_Delete.cs

```diff
@@ -153,15 +156,37 @@ public int DeleteLoadcases(IEnumerable<object> ids)
             }
             else
             {
-                int maxNodeIndex = m_RobotApplication.Project.Structure.Cases.FreeNumber;
-                for (int i = 1; i < maxNodeIndex; i++)
+                caseSel = m_RobotApplication.Project.Structure.Selections.CreatePredefined(IRobotPredefinedSelection.I_PS_CASE_SIMPLE_CASES);               
+            }            
+            m_RobotApplication.Project.Structure.Cases.DeleteMany(caseSel);
+            m_tags[typeof(Bar)] = caseTags;
+            BH.Engine.Reflection.Compute.RecordWarning("Note that when all cases are deleted in Robot, combinations are deleted also");
```

Comment:
think the position is better for this now, but still shows up, even if nothing is deleted. Maybe worth raising the warning only if

`If( ids == null || ids.Count() != 0)`

That is to skip the warning when the ids is a valid IEnumerable, but empty, hence nothing will get removed